webapps/frontend/public/app/admin/locales/de.json

```diff
@@ -1,4 +1,95 @@
 ﻿{
+  "dateLocales": {
```

Comment:
Without `dateLocales` our [locales.js skips initializing moment locales](https://github.com/camunda/camunda-bpm-platform/blob/446c4d0373f898d67f9b371108a9e0c66c6f8b93/webapps/frontend/ui/common/scripts/services/locales.js#L111). We use moment for the months formatting.